src/samples/ConsoleSample/ConsoleSample.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jab" Version="0.11.0" PrivateAssets="all" />
+    <PackageReference Include="Jab" Version="0.12.0" PrivateAssets="all" />
```

Comment:
Can you revert version updates in sample apps please? These are automatically updated after the release.